### PR TITLE
add license file to proxy agent docker image

### PR DESCRIPTION
**Pull request by IronPan**
/assign @hongye-sun

---
This change is [Reviewable](https://reviewable.io/reviews/kubeflow/pipelines/1054)

### CODE REVIEW

File: proxy/Dockerfile [hunk 1]
```diff
@@ -17,7 +17,8 @@ FROM gcr.io/inverting-proxy/agent@sha256:d0a06a247bb443f9528356a1341cadfa4c4479a
```
Comment (hongye-sun) [inline proxy/Dockerfile]: Do you know if the inverting-proxy image has all the licenses?

Comment (IronPan) [reply proxy/Dockerfile]: yes it does have the licenses.

**Pull request merged**